[2023-08-03 03:22:16,687] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.upload_weather_data manual__2023-08-03T03:22:03.288396+00:00 [queued]>
[2023-08-03 03:22:16,695] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.upload_weather_data manual__2023-08-03T03:22:03.288396+00:00 [queued]>
[2023-08-03 03:22:16,695] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-03 03:22:16,695] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-03 03:22:16,695] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-03 03:22:16,722] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): upload_weather_data> on 2023-08-03 03:22:03.288396+00:00
[2023-08-03 03:22:16,727] {standard_task_runner.py:52} INFO - Started process 119 to run task
[2023-08-03 03:22:16,730] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'S3_to_BigQuery', 'upload_weather_data', 'manual__2023-08-03T03:22:03.288396+00:00', '--job-id', '448', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp5b973iuj', '--error-file', '/tmp/tmpxcdocu6s']
[2023-08-03 03:22:16,730] {standard_task_runner.py:80} INFO - Job 448: Subtask upload_weather_data
[2023-08-03 03:22:16,784] {task_command.py:370} INFO - Running <TaskInstance: S3_to_BigQuery.upload_weather_data manual__2023-08-03T03:22:03.288396+00:00 [running]> on host 6c95ca7a45b2
[2023-08-03 03:22:16,876] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=S3_to_BigQuery
AIRFLOW_CTX_TASK_ID=upload_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-03T03:22:03.288396+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-03T03:22:03.288396+00:00
[2023-08-03 03:22:22,779] {gbq.py:606} INFO - 46 out of 46 rows loaded.
[2023-08-03 03:22:22,779] {logging_mixin.py:115} INFO - 
✅ Table weather_topic/2023-08-02 upload on BigQuery! ✅
[2023-08-03 03:22:22,779] {python.py:173} INFO - Done. Returned value was: None
[2023-08-03 03:22:22,790] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=S3_to_BigQuery, task_id=upload_weather_data, execution_date=20230803T032203, start_date=20230803T032216, end_date=20230803T032222
[2023-08-03 03:22:23,001] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-03 03:22:23,052] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
